src/modules/timepix3/geometry/Config.h

```diff
@@ -12,6 +12,7 @@
 #include <common/debug/Trace.h>
 #include <logical_geometry/ESSGeometry.h>
 #include <string>
+#include <sys/types.h>
```

Comment:
Doesn't matter so much, but is this one really necessary?

#include <cinttypes>

Should probably work fine.